deploy/helm/omni/values.yaml

```diff
@@ -0,0 +1,72 @@
+global: {}
+
+omni:
+  nameOverride: omni  # optional, if ommitted the Chart name will be used
+  debug: false
+  domainName: omni.makomedia.nl
+  accountUuid: D5C2A21E-7162-4122-B923-A87ED7827F5B
+  privateKeySource: "/omni.asc"
+  namespace:
+    name: omni
```

Comment:
Namespace should be set via the Helm args (`helm install --namespace ...`) and templated in the chart as `{{ .Release.Namespace }}`.
```suggestion
```

Reply:
Agreed, will change that.